hal/src/msom/pinmap_defines.h

```diff
@@ -61,7 +61,7 @@
 #define A4                  D15
 #define A5                  D14
 #define A6                  D29
-#define A7                  VBAT_MEAS
```

Comment:
Right now `A7` `D28` `WKP` and `VBAT_MEAS` are all the same logical pin. Should we get rid of the `VBAT_MEAS` define altogether? It should not be used by name?

Reply:
Probably only used to refer to manufacturer's pinout.  We can refer to it as A7 Internally.